crystalfontz/cli.py

```diff
@@ -413,7 +523,6 @@ def main(
     model: str,
     hardware_rev: Optional[str],
     firmware_rev: Optional[str],
-    detect: bool,
```

Comment:
Intended workflow is to call `crystalfontz config detect` and then use the updated config.